styles.css

```diff
@@ -312,7 +312,7 @@ body {
   margin-bottom: 28px;
   background: #FD4766;
   background: linear-gradient(120deg, #1C99FE 20.69%, #7644FF 50.19%, #FD4766 79.69%);;
-  -webkit-background-clip: text;
+

```

Comment:
@kairveeehh there is a problem 

![image](https://github.com/apu52/Travel_Website/assets/114172928/c3e93bd5-ce1f-4665-aab5-f914f92b1851)

The Headings are not visible